src/Rubric-Tests/RubEditingAreaTest.class.st

```diff
@@ -59,6 +59,31 @@ RubEditingAreaTest >> setUp [
 	area setTextWith: 'one two three four'.
 ]
 
+{ #category : #'tests - accessing selection' }
```

Comment:
no dots at end of method necessary. There is nothing that should follow tearDown

Reply:
I know - they are not warned against, and when I edit a method they sometimes end up being left behind. Just like the extra spaces at the end of a line which are (were) inserted by the completion mechanism. It would be really useful to have an editor option to remove unnecessary dots and blanks at accept.

Reply:
Yes - it is already on my "cleanup keep/improve code quality level" TODO list